packages/core/src/useTextField/useTextControl.ts

```diff
@@ -0,0 +1,64 @@
+import { shallowRef, toValue } from 'vue';
+import { InputEvents, Reactivify } from '../types';
+import { propsToValues, useCaptureProps } from '../utils/common';
+import { TextControlProps } from './types';
+import { useInputValidity } from '../validation';
+import { FieldState, useFieldStateInjection } from '../useFieldState';
+import { FormField, useFormFieldInjection } from '../useFormField';
+
+interface FormControlContext {
+  inputId: string;
+  field?: FormField;
+  state?: FieldState<string | undefined>;
+}
+
+export function useTextControl(props: Reactivify<TextControlProps>, ctx?: FormControlContext) {
+  const inputEl = shallowRef<HTMLInputElement>();
+  const field = ctx?.field ?? useFormFieldInjection();
+  const state = ctx?.state ?? useFieldStateInjection<string | undefined>();
+
+  if (state) {
+    useInputValidity({ inputEl, field: state, disableHtmlValidation: props.disableHtmlValidation });
```

Comment:
In the current behavior in Nuxt ui, there is real-time validation _**AFTER**_ the initial blur.

Are we able to achieve something similar with the current formwerk API. It looks like `useInputValidity()` does accept a list of events.

Reply:
Yep, you can send in the events you need. By default its `blur` and `change`.